catalog/core/catalog-core-commons/src/main/java/ddf/util/XPathHelper.java

```diff
@@ -156,12 +158,20 @@ public static String print(Node n, String encoding) {
       StringWriter stringOut = new StringWriter();
 
       DOMImplementationLS domImpl = (DOMImplementationLS) document.getImplementation();
-      LSSerializer serializer = domImpl.createLSSerializer();
-      LSOutput lsOut = domImpl.createLSOutput();
-      lsOut.setEncoding(encoding);
-      lsOut.setCharacterStream(stringOut);
+      LSSerializer lsSerializer = domImpl.createLSSerializer();
+      lsSerializer.getDomConfig().setParameter(XML_DECLARATION, false);
```

Comment:
❓ Was this required for the attribute serialization to work properly?

Reply:
Not required. Through discussions with @shaundmorris, when using node in xpathhelper.evaluate/.print it will return the data with the xml declaration and we believed it better if it didn't.